package.json

```diff
@@ -1,6 +1,18 @@
 {
+  "name": "ve-governance",
+  "version": "1.0.0",
   "devDependencies": {
     "prettier": "^3.3.3",
     "prettier-plugin-solidity": "^1.4.1"
+  },
+  "scripts": {
+    "build": "forge build",
+    "build-clean": "forge build --force",
+    "build-refs": "bash script/build-reference-builds.sh",
+    "build-all": "bun run build-refs && bun run build-clean",
+    "test-unit": "forge test --no-match-path \"{test/fork/*.sol,test/upgrades/*.sol}\" -vvv",
+    "test-fork": "forge test --match-path \"test/fork/**/*.sol\" --rpc-url https://sepolia.drpc.org -vvv",
+    "test-upgrades": "forge test --build-info --match-path \"test/upgrades/**/*.sol\" -vvv",
+    "test": "forge test -vvv"
```

Comment:
these should be added to the makefile or we replace the makefile

Reply:
100%, better all together. And make is easier to use in CI